### PR TITLE
[update]: update Exchange page UI to mockup

### PR DESCRIPTION
**Pull request by dhruv1294**
**UI** : Exchange page and exchange bottom sheet now looks same as the figma mockup.

**TODOS**:  

-  Text input spinbox for mobile.
-  loading svg assets for mobile
-  transition animation for bottom sheet

### CODE REVIEW

File: lib/components/sheets/exchange_bottom_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+class _ExchangeBottomSheetState extends State<ExchangeBottomSheet>
+    with TickerProviderStateMixin {
+  @override
+  Widget build(BuildContext context) {
+    int priceChange =
+        (widget.company.currentPrice - widget.company.previousDayClose).toInt();
+    int quantity = 1;
+    int totalPrice = widget.company.currentPrice.toInt() * quantity;
+    int orderFee = _calculateOrderFee(totalPrice);
+    return BlocProvider(
```
Comment (Sudhindra3000) [inline lib/components/sheets/exchange_bottom_sheet.dart:34]: Can you also split up this widget tree like in ExchangePage

File: lib/components/sheets/exchange_bottom_sheet.dart [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+                                            ),
+                                          ],
+                                        ),
+                                      ]),
+                                  Row(
+                                      mainAxisAlignment:
+                                          MainAxisAlignment.start,
+                                      children: [
+                                        SvgPicture.asset(
+                                          '/icon/balance.svg',
```
Comment (satyamurti) [inline lib/components/sheets/exchange_bottom_sheet.dart:304]: Create a file called `icons.dart` in `constants/` directory and add this string there, refer to trading_dashboard branch.

**Review by satyamurti — CHANGES_REQUESTED** [review]

File: lib/components/sheets/exchange_bottom_sheet.dart (hunk 2 above)
Comment (dhruv1294) [reply lib/components/sheets/exchange_bottom_sheet.dart:304]: also need to find a fix for loading svg in mobile

Comment (satyamurti) [reply lib/components/sheets/exchange_bottom_sheet.dart:304]: we can have meet and resolve it

**Review by Sudhindra3000 — APPROVED** [review]

**Review by satyamurti — APPROVED** [review]

**Pull request merged**